pkg/util/helpers.go

```diff
@@ -40,6 +41,31 @@ type Container struct {
 	PreStopConfig         *horizonapi.ActionConfig
 }
 
+// PodAffinityConfig used for Pod and AntiPod Affinity Configuration
+type PodAffinityConfig struct {
+	Type   horizonapi.AffinityType
+	Config horizonapi.PodAffinityConfig
+}
+
+// NodeAffinityConfig used for Node Affinity Configuration
+type NodeAffinityConfig struct {
+	Type   horizonapi.AffinityType
+	Config horizonapi.NodeAffinityConfig
+}
+
+// PodConfig used for configuring the pod
+type PodConfig struct {
+	Labels                 map[string]string
+	ServiceAccount         string
+	Containers             []*Container
+	Volumes                []*components.Volume
+	InitContainers         []*Container
+	PodAffinityConfigs     []*PodAffinityConfig
```

Comment:
Instead of another structure to wrap 2 horizon apis, why not just make this a map[AffinityType][]PodAffinityConfig?  There would then also be no need to fill it in for pods that don't have affinity config because range over an empty map won't do anything.

Reply:
done